tests/lambda_endpoints/test_ialirt_ingest.py

```diff
@@ -11,20 +10,22 @@ def populate_table(table):
     """Populate DynamoDB table."""
     items = [
         {
+            "apid": 478,
             "met": 123,
-            "ingest_time": "2021-01-01T00:00:00Z",
+            "ingest_date": "2021-01-01T00:00:00Z",
             "packet_blob": b"binary_data_string",
         },
         {
+            "apid": 478,
             "met": 124,
-            "ingest_time": "2021-01-01T00:00:01Z",
+            "ingest_date": "2021-02-01T00:00:00Z",
             "packet_blob": b"binary_data_string",
         },
     ]
     for item in items:
         table.put_item(Item=item)
```

Comment:
No need to fix this here, but there is a batch_write_item that I think will do multiple inserts at once.
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/dynamodb/client/batch_write_item.html